common/c_cvarlist.cpp

```diff
@@ -227,6 +227,9 @@ CVAR(g_preroundtime, "5", "Amount of time before a round where you can't shoot",
 CVAR(g_postroundtime, "3", "Amount of time after a round before the next round/endgame",
      CVARTYPE_INT, CVAR_SERVERARCHIVE | CVAR_NOENABLEDISABLE)
 
+CVAR_RANGE(g_coopthingfilter, "0.0",
+     "Removes cooperative things of the map.\nValues: 1 - Only Coop weapons.\n 2 - All Coop things.",
```

Comment:
As written, if this gets written into a config file, it will break parsing of said config file.  You need to follow every `\n` with `//` so it shows up as a comment.

Reply:
Last commit should fix it. I actually forgot about those newlines.

Reply:
Cool I think we're golden.